Sitenotice.php

```diff
@@ -23,7 +23,7 @@ function onSiteNoticeAfter( &$siteNotice, $skin ) {
 
 		$siteNotice .= <<<EOF
 			<table class="wikitable" style="text-align:center;"><tbody><tr>
-			<td style="font-size:125%"><div data-nosnippet>Miraheze has upgraded to MediaWiki 1.36.0. If you encounter any technical issues, please create a <a href="https://phabricator.miraheze.org" target="_blank">Phabricator</a> task.</div></td>
+			<td style="font-size:125%"><div data-nosnippet>Miraheze has upgraded to MediaWiki 1.36.0. If you encounter any technical issues, please create a <a href="https://phabricator.miraheze.org" target="_blank">Phabricator</a> task. Additionally and unfortunately, Unfortunately, Miraheze has had to temporarily disable the <a href="https://www.mediawiki.org/wiki/Extension:DiscussionTools" target="_blank">DiscussionTools</a> extension due a technical issue with compatibility. A patch has been submitted for review, and it will be reenabled as soon as possible. Thank you</div></td>
```

Comment:
and unfortunately, Unfortunately does not sound right.

Reply:
LOL, yeah, I just noticed that at the same time you commented. Copy/paste error on my part. Should be fixed with https://github.com/miraheze/mw-config/pull/3957/commits/a4539551f4a9e0ee7776faaae5d10d208fe30a36